packages/react-core/src/components/ApplicationLauncher/__tests__/__snapshots__/ApplicationLauncher.test.tsx.snap

```diff
@@ -98,7 +98,6 @@ exports[`ApplicationLauncher custom icon 1`] = `
     isPlain={false}
     menuAppendTo="inline"
     onSelect={[Function]}
-    ouiaComponentType="Dropdown"
```

Comment:
This is wrong - I know we removed one of the explicit references to the string name within `Dropdown`, can you point me to the fix for this please?

Reply:
This is showing the line as removed, which is what I assume would happen when "we removed one of the explicit references to the string name within `Dropdown`"